packages/react/src/components/Card/Card.jsx

```diff
@@ -516,7 +516,7 @@ const Card = (props) => {
                 triggerClassName={`${iotPrefix}--card--header--tooltip`}
                 id={`card-tooltip-${id}`} // https://github.com/carbon-design-system/carbon/pull/6744
                 triggerText=""
-                iconDescription={`card-header-tooltip-${id}`} // To fix accessibility violation.
+                iconDescription={strings.iconDescription || `card-header-tooltip-${id}`} // To fix accessibility violation.
```

Comment:
We should add this prop to the packages/react/src/constants/CardPropTypes.js and perhaps call it titleTooltipIconDescription and also give that prop a default value like "Tooltip info icon".

Reply:
And since we now merged https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3402 we need to move these changes to the new CardTitle component and also include this new iconDescription prop